pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -1971,6 +1971,14 @@ func (ws *WorkflowStatus) TaskResultsInProgress() bool {
 	return false
 }
 
+func (ws *WorkflowStatus) IsTaskResultInCompleted(name string) bool {
```

Comment:
assuming we keep this method, let's rename it to `IsTaskResultIncomplete()` (current name sounds a bit like the opposite)

Reply:
ok，I will rename it.

Reply:
Actually, to match the name above, it should probably be `IsTaskResultInProgress()`